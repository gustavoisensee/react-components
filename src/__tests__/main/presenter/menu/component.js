import React from 'react';
import renderer from 'react-test-renderer';
import PresenterMenu from './../../../../modules/main/presenter/menu/component';

describe('Main/presenter/menu component test', () => {
  test('Main/presenter/menu component renders', () => {
    const props = {
      menu: [],
      changeMenuItem: () => {}
    };

    const tree = renderer.create(
      <PresenterMenu {...props} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
