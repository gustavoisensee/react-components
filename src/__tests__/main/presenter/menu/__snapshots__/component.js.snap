exports[`Main/presenter/menu component test Main/presenter/menu component renders 1`] = `
<div
  className="container">
  <div
    className="title">
    <span>
      Test Menu
    </span>
  </div>
  <div
    className="menu">
    <div
      className="container" />
  </div>
  <div
    className="description">
    Description
  </div>
</div>
`;
